config/hack/CONTAINER_IMAGES

```diff
@@ -12,5 +12,12 @@
     "namespace": "gaudi-metric-exporter",
     "name": "metric-exporter",
     "tag": "1.6.0-439"
+  },
+  {
+    "tplvar": "NODE_LABELER_IMAGE",
+    "registry": "ghcr.io",
+    "namespace": "mwalter-habana",
+    "name": "habana-node-labeler",
+    "tag": "0.0.1"
```

Comment:
After our offline communication with @mwalter-habana , this tag should be updated to `0.0.2`.

#### Context

In AWS the naming defers, instead of `Gaudi` it's listed as `Gaudi secured`, which introduced a validation error to the NFD labels generated. The latter included a space, i.e. `habana.ai/hpu.gaudi secured.present=true`, which is invalid.  

```shell
# AWS dl1.24xlarge instance
$ lspci
00:00.0 Host bridge: Intel Corporation 440FX - 82441FX PMC [Natoma]
00:01.0 ISA bridge: Intel Corporation 82371SB PIIX3 ISA [Natoma/Triton II]
00:01.3 Non-VGA unclassified device: Intel Corporation 82371AB/EB/MB PIIX4 ACPI (rev 08)
00:03.0 VGA compatible controller: Amazon.com, Inc. Device 1111
00:04.0 Non-Volatile memory controller: Amazon.com, Inc. NVMe EBS Controller
10:00.0 Ethernet controller: Amazon.com, Inc. Elastic Network Adapter (ENA)
10:1d.0 Processing accelerators: Habana Labs Ltd. HL-2000 AI Training Accelerator [Gaudi secured] (rev 01)
10:1e.0 Processing accelerators: Habana Labs Ltd. HL-2000 AI Training Accelerator [Gaudi secured] (rev 01)
10:1f.0 Non-Volatile memory controller: Amazon.com, Inc. NVMe SSD Controller
20:1d.0 Processing accelerators: Habana Labs Ltd. HL-2000 AI Training Accelerator [Gaudi secured] (rev 01)
20:1e.0 Processing accelerators: Habana Labs Ltd. HL-2000 AI Training Accelerator [Gaudi secured] (rev 01)
20:1f.0 Non-Volatile memory controller: Amazon.com, Inc. NVMe SSD Controller
90:1d.0 Processing accelerators: Habana Labs Ltd. HL-2000 AI Training Accelerator [Gaudi secured] (rev 01)
90:1e.0 Processing accelerators: Habana Labs Ltd. HL-2000 AI Training Accelerator [Gaudi secured] (rev 01)
90:1f.0 Non-Volatile memory controller: Amazon.com, Inc. NVMe SSD Controller
a0:1d.0 Processing accelerators: Habana Labs Ltd. HL-2000 AI Training Accelerator [Gaudi secured] (rev 01)
a0:1e.0 Processing accelerators: Habana Labs Ltd. HL-2000 AI Training Accelerator [Gaudi secured] (rev 01)
a0:1f.0 Non-Volatile memory controller: Amazon.com, Inc. NVMe SSD Controller
```